k8s/openebs-operator.yaml

```diff
@@ -315,9 +315,10 @@ spec:
         livenessProbe:
           exec:
             command:
-            - pgrep
-            - -f
-            - ".*openebs"
+            - sh
+            - -c
```

Comment:
how about ```test `pgrep -c ^openebs-provisi.*` == 1```

Reply:
changed to `test `pgrep -c ^openebs-provisi.*` = 1` ( test support single `=` expression example string1 = string2 )